package.json

```diff
@@ -38,17 +38,22 @@
       "js"
     ],
     "moduleNameMapper": {
-      "^@taquito/rpc$": "<rootDir>/packages/taquito-rpc/src/taquito-rpc.ts",
-      "^@taquito/taquito$": "<rootDir>/packages/taquito/src/taquito.ts",
+      "^pack-test-tool$": "<rootDir>/packages/taquito-michel-codec/pack-test-tool/src/pack-test-tool.js",
+      "^@taquito-beacon-wallet$": "<rootDir>/packages/taquito-beacon-wallet/src/taquito-beacon-wallet.ts",
```

Comment:
what was the reasoning behind the missing moduleNameMapper entries again?

Reply:
I'm not completely sure. Could not find good resources. But from my understanding, all workspace modules that can be used by other modules are expected to be added here.